### PR TITLE
Unittesting

### PR DESCRIPTION
**Pull request by manthinaa**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -11,6 +12,7 @@ dependencies {
     compile 'org.apache.directory.studio:org.dom4j.dom4j:1.6.1'
     compile 'log4j:log4j:1.2.17'
     compile 'org.apache.httpcomponents:httpclient:4.3.5'
+    compile 'javax.servlet:javax.servlet-api:3.1.0'
```
Comment (michaelrice) [inline build.gradle]: Hey Anil, what exactly is this needed for?

File: src/main/java/com/vmware/vim25/ws/SoapClient.java [hunk 2]
```diff
@@ -174,4 +177,38 @@ public StringBuffer readStream(InputStream is) throws IOException {
...
+     */
+    public Object unMarshall(String returnType, InputStream is) {
+        Object unMarshalledObj = null;
+        try {
+            unMarshalledObj = xmlGen.fromXML(returnType, is);
+        } catch (Exception e) {
+            log.error(
+                    "Exception Occured while unmarshalling the inputstream to java object of type:"
+                            + returnType, e);
+            e.printStackTrace();
```
Comment (michaelrice) [inline src/main/java/com/vmware/vim25/ws/SoapClient.java]: @manthinaa Can you remove this print?

Comment (manthinaa) [inline src/main/java/com/vmware/vim25/ws/SoapClient.java]: Fixed

File: build.gradle (hunk 1 above)
Comment (manthinaa) [inline build.gradle]: Fixed

**Pull request merged**